packages/credentials/src/issuer.ts

```diff
@@ -15,6 +15,16 @@ import type { CTypeLoader } from './ctype/index.js'
 import type { IssuerOptions, SubmitOverride } from './interfaces.js'
 
 export type { IssuerOptions }
+import * as Kilt from '@kiltprotocol/sdk-js'
+import * as KiltChain from '@kiltprotocol/chain-helpers'
```

Comment:
As with the other PR, no imports from sdk-js here and no blanket imports.

Reply:
Done at: https://github.com/KILTprotocol/sdk-js/pull/918/commits/ccac9b90e1ceac14a1c26d443d3fcef994745055